docs/jaeger-support.asciidoc

```diff
@@ -133,6 +155,19 @@ Defaults to the standard Jaeger gRPC collector port `14250`.
 Set to the name of the tag that should be used for authorizing Jaeger agents.
 By default, authorization does not apply to Jaeger agents.
 
+[float]
+===== `grpc.sampling.enabled`
+Enable the sampling rate to be fetched from the gRPC endpoint.
+Only probabilistic sampling is supported.
+No authorization token can be configured for this endpoint.
+Sampling is disabled by default.
+
+[float]
+===== `grpc.sampling.default`
+Set the default sampling rate used when no service specific sampling is configured.
+Valid range is from `0` to `1.0`, where `0` samples at 0% and `1.0` samples at 100%.
+Defaults to `1.0`.
```

Comment:
Sorry for the additional work here, as specified above, I removed the config options again.

Reply:
No problem! That's what I get for starting early on it 😁